graphistry/falkordb_util.py

```
@@ -0,0 +1,198 @@
+    :return: FalkorDB instance or None
+    """
+    if db is None:
+        return None
+    try:
+        from falkordb import FalkorDB
+        if isinstance(db, FalkorDB):
+            return db
+        # If dict is provided, create FalkorDB instance
+        return FalkorDB(**db)
+    except ImportError:
+        raise FalkorDBSupportModuleNotFound()
```

Ruff (0.14.4)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)